sources/lib/datalab/gcp/datalab/_utils.py

```diff
@@ -57,7 +65,12 @@ def get_field_list(fields, schema):
 
 
 def _get_cols(fields, schema):
-  """ Get column metadata for Google Charts based on field list and schema. """
+  """ Get column metadata for Google Charts based on field list and schema.i
```

Comment:
Done. This is what happens when you put a non-vi editor into vi-emulation! It doesn't always start in command mode.